### PR TITLE
logging msgs added

### PR DESCRIPTION
**Pull request by kvoytikh**
logging msgs added

### CODE REVIEW

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
+public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
+    @Override
+    public Order completeOrder(Long userId) {
+        logger.info("completeOrder method was called. Params: userId {}", userId);
+        List<Product> products = getAllProductsFromShoppingCart(userId);
+        Order order = new Order(products, userId);
+        // NOTE: In production ready code this order identifier should be generated by DB
+        // For test purpose we simplify this and return dummy data
```
Comment (Radikhovskiy) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:19]: Comments should be deleted

File: jv-logger.iml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <excludeFolder url="file://$MODULE_DIR$/src/main/java/resources" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Maven: org.apache.logging.log4j:log4j-core:2.14.1" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.logging.log4j:log4j-api:2.14.1" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: junit:junit:4.12" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-core:1.3" level="project" />
+  </component>
+</module>
```
Comment (Radikhovskiy) [inline jv-logger.iml:19]: Here it may doesn't matter but need a new line

**Review by Radikhovskiy — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-[Try to avoid these common mistakes while solving task](https://mate-academy.github.io/jv-program-common-mistakes/java-core/logger/logger)
+# Logger
+
+1. Configure logger in this project
+1. Edit `.gitignore` file so that your file with logs is NOT pushed to Github
+1. If you have `System.out.println()` then replace them all with corresponding logger methods
+1. Complete all `TODO` statements
+1. Replace all `e.printStackTrace();`  with corresponding logger methods
+
+[Try to avoid these common mistakes while solving task](https://mate-academy.github.io/jv-program-common-mistakes/java-core/logger/logger)
```
Comment (Max-Pochepets) [inline README.md:9]: remove this from PR

File: jv-logger.iml (hunk 2 above)
Comment (Max-Pochepets) [reply jv-logger.iml:19]: ++

File: src/main/java/mate/academy/Main.java [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
+public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        AuthenticationService authenticationService = new AuthenticationServiceImpl();
+        User user;
+        try {
+            user = authenticationService.login("bob", "1234");
+        } catch (AuthenticationException e) {
+            logger.error("Method login doesn't work successfully", e);
```
Comment (Max-Pochepets) [inline src/main/java/mate/academy/Main.java:21]: this is not informative enough. need to specify what happened and why method failed

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
why all files look like written from the scratch. let's clean this PR

**Comment by kvoytikh** [comment]
fixed msg

**Pull request closed without merging**